SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -201,8 +202,17 @@ public SingularityRequestParent bounce(@ApiParam("The request ID to bounce") @Pa
 
     requestManager.bounce(requestWithState.getRequest(), System.currentTimeMillis(), JavaUtils.getUserEmail(user), message);
 
+    SingularityBounceRequest defaultBounceRequest = bounceRequest.or(SingularityBounceRequest.defaultRequest());
+    if (!defaultBounceRequest.getDurationMillis().isPresent()) {
+      final Long durationMillis = TimeUnit.MINUTES.toMillis(configuration.getDefaultBounceExpirationMinutes());
```

Comment:
^ 👍. The JVM's ability to hop between primitives (`long`) and "boxed" types (`Long`) is called auto-boxing. This is just one of those style things everyone learns the hard way 😃